saleor/discount/utils.py

```diff
@@ -599,9 +600,10 @@ def prepare_line_discount_objects_for_catalogue_promotions(lines_info):
     )
 
 
-def _get_discount_amount(
-    variant_channel_listing: "ProductVariantChannelListing", line_quantity: int
-) -> Decimal:
+def _is_discounted_line(
```

Comment:
What do you think about extending the name to `_is_discounted_line_by_catalogue_promotion` or something similar that will give more context that this function checks only catalogue promotions. 
Just to be as explicit as possible. (also, I don't think that we need to port this to the previous version - as I saw that we already merged this to 3.19. It is only visual improvement)